common.h

```diff
@@ -501,14 +504,16 @@ RK_ORDER2_STEP1,   /*!< Two-step second order method, first step */
 RK_ORDER2_STEP2    /*!< Two-step second order method, second step */
 };
 
-const int WID = 4;         /*!< Number of cells per coordinate in a velocity block. Only a value of 4 supported by vectorized Leveque solver */
+#ifndef WID
+#define WID (4)              /*!< Number of cells per coordinate in a velocity block. Defaults to the historical 4. */
```

Comment:
How (un)likely is it that some system/library/<...> uses an env variable called this which might interfere?

Reply:
Would you prefer that this read:
```
#ifndef VLASIATOR_WID
#define VLASIATOR_WID (4)
#endif
const int WID = VLASIATOR_WID;
```
(assuming it works?)

Reply:
I think it's fine as it's been, it never interfered.